presets/ragengine/vector_store/base.py

```diff
@@ -103,6 +105,7 @@ async def _create_index_common(self, index_name: str, documents: List[Document],
                         storage_context=storage_context,
                         embed_model=self.embed_model,
                         use_async=True,
+                        transformations=[self.custom_transformer],
```

Comment:
cool, I assume transformation gets applied every time new document comes in as well